translation/source/variantShortName.xml

```diff
@@ -34,8 +34,10 @@
   <string name="flipello10">XFlip</string>
   <string name="amazons">Amazons</string>
   <string name="oware">Oware</string>
-  <string name="togyzkumalak">Togyz</string>
+  <string name="togyzkumalak">Toqyz</string>
   <string name="go9x9">Go9</string>
   <string name="go13x13">Go13</string>
   <string name="go19x19">Go19</string>
+  <string name="backgammon">Backgammon</string>
```

Comment:
I think these are slightly too long by comparison to the others. Suggest Backg and Nackg?